app/models/trainee.rb

```diff
@@ -235,6 +235,7 @@ class Trainee < ApplicationRecord
   before_save :clear_lead_school_id, if: :lead_school_not_applicable?
   before_save :set_submission_ready, if: :completion_trackable?
 
+  after_commit :update_trainee_in_dqt, on: :update, if: :dqt_updatable?
```

Comment:
I wonder it's worth having an exclude list for certain field updates, similar to the one we have below in the value_digest? 🤷‍♀️

Reply:
> We probably will end up needing something. I'm trying to avoid having too much DQT business logic as to what updates they're interested in. Ideally we'd just send on any update and they would decide whether they're interested or not. @gunndabad any thoughts?

For now I think that's ok. If the changes end up generating too much noise for Rob's team we may have to review.